src/styles/normalize.css

```
@@ -0,0 +1,17 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+p {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+
+  font-family: Roboto, Times, serif;
```

Roboto is a sans-serif font, use Arial and sans-serif as fallback fonts
```suggestion
  font-family: Roboto, Arial, sans-serif;
```